requirements.txt

```diff
@@ -1,3 +1,4 @@
+git+https://github.com/ClusterHQ/pyrsistent@5b722c3ad75ed2da414a40e6d27af373a8610018#egg=pyrsistent-0.11.13chq
```

Comment:
This PR is built on top of https://github.com/ClusterHQ/flocker/pull/2725

Once that is merged the only change in this PR should be the hash in this line.